src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/Components/TCEvaluationTable/TCEvaluationRows/TCEvaluationCells/index.js

```diff
@@ -60,14 +60,14 @@ class TCEvaluationCells extends Component {
     if(this.props.value <= 0.2){
       return (
         <div className="TCEvaluationCellsPercentShadedWarning">
-          <img src={warning} height={16}/>
+          <img src={warning} height={16} alt="Can't Find"/>
```

Comment:
Here, the alt text should reflect the purpose of the image in the UI -- let's use "Warning" as the text for this one.